org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/coloring/IColoringService.java

```diff
@@ -48,7 +46,7 @@
 
 		@Override
 		public ColoringParams getColoring(final XtextResource resource, final Document document) {
```

Comment:
Should the service only return a `List<? extends ColoringInformation>`? I hink the Params and URI should be set outside.